todo_app/app.py

```diff
@@ -1,15 +1,29 @@
-from flask import Flask
-
+from types import GetSetDescriptorType
+from flask import Flask,escape,request,render_template,redirect
+from flask.helpers import url_for
+from flask.wrappers import Request
+from todo_app.data.session_items import _DEFAULT_ITEMS, add_item,get_items
 from todo_app.flask_config import Config
 
 app = Flask(__name__)
 app.config.from_object(Config)
 
-
-@app.route('/')
+@app.route('/', methods=['GET', 'POST'])
 def index():
-    return 'Hello World!'
+    if request.method == "POST":
+       return redirect(url_for('addItm'))
+    else:
+        return render_template('index.html',items=get_items())
```

Comment:
This works nicely here, but as these routes could potentially grow in complexity, you'll probably find it nicer to separate these out into separate functions, e.g.
```python
@app.route('/', methods=['GET'])
def index():
    return render_template('index.html',items=get_items())

@app.route('/', method=['POST'])
def postIndex():
    return redirect(url_for('addItm'))
```

Having said that, given that the POST endpoint just redirects to the relevant page, it would be clearer to have that just be a link that the user clicks so we could lose the POST part of this method entirely